JYH/7Day/모의고사.js

```diff
@@ -0,0 +1,28 @@
+function solution(answers) {
+    let person1 = [1, 2, 3, 4, 5];
+    let person2 = [2, 1, 2, 3, 2, 4, 2, 5];
+    let person3 = [3, 3, 1, 1, 2, 2, 4, 4, 5, 5];
+    let arr = [0, 0, 0];
+    let result =[];
+    let length1 = person1.length, length2 = person2.length, length3 = person3.length;
```

Comment:
```suggestion
    const person1 = [1, 2, 3, 4, 5];
    const person2 = [2, 1, 2, 3, 2, 4, 2, 5];
    const person3 = [3, 3, 1, 1, 2, 2, 4, 4, 5, 5];
    const arr = [0, 0, 0];
    const result =[];
    const length1 = person1.length;
    const length2 = person2.length;
    const length3 = person3.length;
```
@jungyoonho128 https://github.com/modern-agile-team/9term-assignment/pull/38/files#r1876351432
해당 코멘트 읽어보셨을까요?